server/src/internalClusterTest/java/org/elasticsearch/search/sort/FieldSortIT.java

```diff
@@ -524,6 +524,9 @@ public void testSimpleSorts() throws Exception {
                     .startObject("float_value")
                     .field("type", "float")
                     .endObject()
+                    .startObject("half_float_value")
```

Comment:
my initial concern was if this integration test would ever be multi-node. This way we have multiple data nodes and thus might sort again on the coordinator. But it looks like this is randomly multiple data nodes. So I think we are good on coverage there.